wls-auth-service/src/main/java/de/muenchen/oss/wahllokalsystem/authservice/domain/OAuthServerSession.java

```diff
@@ -0,0 +1,23 @@
+package de.muenchen.oss.wahllokalsystem.authservice.domain;
+
+public class OAuthServerSession {
+
+    private String username;
+    private String sessionId;
+
+    public String getUsername() {
+        return username;
+    }
+
+    public void setUsername(String username) {
+        this.username = username;
+    }
```

Comment:
this proposal is outdated since we changed the Class acording to an older suggestion of you, removed the explicit set and get methods and annotated it with @Data